tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -644,6 +645,8 @@ bool FilterPipeline::use_tile_chunking(
     } else if (version >= 13 && has_filter(FilterType::FILTER_DICTIONARY)) {
       return false;
     }
+  } else if (has_filter(FilterType::FILTER_WEBP)) {
```

Comment:
If an array is already written using `FILTER_WEBP` with chunking enabled, will it be possible to read it after disabling it? Or we need to add a check on the version as well to make sure this applies from now on?
If yes is there a way to fix this problem for older arrays by just adapting the reader, or it's a Writer bug that we are stuck with?

Reply:
Yep good call - I added a test that writes a tile with chunking enabled, then disables chunking prior to reading it back. It looks like we load tile chunk data from the result tiles though, so if we write a tile using chunking it will always be read using chunking. So turns out the test I added is not very interesting for this case, but it's still a stake in the ground for moving webp unit tests out of tiledb_unit so I left it in this PR.

https://github.com/tiledb-inc/tiledb/blob/f6809293d55406ca20703b54a8cfcc48e316ad68/tiledb/sm/query/readers/reader_base.cc#L753

IIUC this means we are ok if a subsequent write will not use chunking, but @kiterluc correct me if I missed something. If you have any ideas on how I could add better testing for this specific edge case I can add them.

Reply:
Thanks for trying that! This means the issue cannot be fixed for arrays already created that use WebP filter, right? Let's highlight that limitation in the bug and PR description for future reference.

Reply:
Done, thanks!